tests/tests_unit/test_data_classes/test_functions.py

```diff
@@ -90,24 +69,6 @@ def test_update_on_deleted_function(self, empty_function):
         empty_function._cognite_client.functions.retrieve.return_value = None
         empty_function.update()
 
-    @pytest.mark.parametrize("limit", [2, None, -1, float("inf")])
-    def test_list_schedules(self, function, function_schedules, limit):
```

Comment:
This test was literally enforcing the bad logic!

Reply:
Getting rid of it, not point unit testing a one line wrapper that is already tested in integration.

Reply:
Agreed 👍👍